storm-core/test/clj/backtype/storm/supervisor_test.clj

```diff
@@ -261,8 +260,8 @@
                                 (str "-Dstorm.id=" mock-storm-id)
                                 (str "-Dworker.id=" mock-worker-id)
                                 (str "-Dworker.port=" mock-port)
-                               "-Dstorm.log.dir=/logs"
-                               "-Dlog4j.configurationFile=/log4j2/worker.xml"
+                                (str "-Dstorm.log.dir=" file-path-separator "logs")
+                                (str "-Dlog4j.configurationFile=" file-path-separator "log4j2" file-path-separator "worker.xml")
```

Comment:
I think you maybe need  fix -Dlog4j.configurationFile as supervisor.clj’s:

          log4j-configuration-file (str (if (.startsWith (System/getProperty "os.name") "Windows")
                                          (if (.startsWith storm-log4j2-conf-dir "file:")
                                            storm-log4j2-conf-dir
                                            (str "file:///" storm-log4j2-conf-dir))
                                          storm-log4j2-conf-dir)
                                     Utils/FILE_PATH_SEPARATOR "worker.xml")